device/linkstate/teprofile.yaml

```diff
@@ -0,0 +1,76 @@
+components:
+  schemas:
+    LinkState.TE:
+      description: >-
+        A container for Traffic Engineering properties on a interface.
+      type: object
+      properties:
+        administrative_group:
+          description: |-
+            The Administrative group sub-TLV (sub-TLV 3). It is a 4-octet 
+            user-defined bit mask used to assign administrative group numbers 
+            to the interface, for use in assigning colors and resource classes. 
+            Each set bit corresponds to a single administrative group for this 
+            interface. The settings translate into Group numbers, which range 
+            from 0 to 31 (integers).
+          type: string
+          format: hex
+          default: '00000000'
+        metric_level:
+          description: |-
+            The user-assigned link metric for Traffic Engineering.
+          type: integer
+          default: 0
+        max_bandwith:
+          description: |-
+            The maximum link bandwidth (sub-TLV 9) in bytes/sec allowed for this 
+            link in this direction.
+          type: integer
+          default: 125000000
+        max_reservable_bandwidth:
+          description: |-
+            The maximum link bandwidth (sub-TLV 10) in bytes/sec allowed for this 
+            link in this direction.
+          type: integer
+          default: 125000000
+        priority_0_bandwith:
```

Comment:
For all the "priority_x_bandwidth" properties, the unit and min/max are not specified.

Also, are these better represented as a vector of fixed size?  Might be easier from an end user standpoint.  I don't know if we have a type like that in OTG, but worth discussing with Andy.

Reply:
@ajbalogh how can we define the properties priority_0_bandwith to priority_7_bandwith in teprofile.yaml in better way. something like priority_0to8_bandwiths[8], a fixed  size array/vector and each entries can be initialized with a same value.

Reply:
This is a possible solution. 
- Note the minimum, maximum values are just suggestions as I don't know what they actual values are.
- Will have to add a test case in `openapiart` to ensure the tooling can accept those keywords.
- Will also have to test `opensource generators` to ensure they can accept those keywords.
```yaml
properties:
  priority_bandwith_0_7:
    type: array
    minItems: 8
    maxItems: 8
    items:
      type: integer
      minimum: 0
      maximum: 125000000
    default: [125000000, 125000000, 125000000, 125000000, 125000000, 125000000, 125000000, 125000000]
```

Reply:
This does not work as expected  but found similar problem like priorities in "layer1 -> flow_control -> Ieee8021qbb -> pfc_class*"
https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/open-traffic-generator/models/master/artifacts/openapi.yaml

We can incorporate that way.